### DISCUSSION TITLE
How do I uninstall?

### DISCUSSION THREAD

**Discussion by Cynthetics** [n00mkrad/flowframes#149] [Q&A] [answered]
Sorry if this is obvious, I have very little knowledge in programming,
How do I uninstall Flowframes on windows 10? My computer crashed almost immediately after I installed it, and it has left the app messed up somehow, I can't find the app on control panel, and I can't uninstall the executable file. There is no uninstaller, only the exe and Flowframesdata.

I want to reinstall it, is it safe to delete the entire folder? Or should I do anything else.

**Comment by DrPleaseRespect** [comment, marked as answer]
Flowframes is a portable program. It doesn't add anything to the control panel, shortcuts for the start menu, mess with the registry.. there's no need for an uninstaller.. just delete the folder...
